src/main/kotlin/it/pagopa/wallet/domain/applications/Application.kt

```diff
@@ -0,0 +1,11 @@
+package it.pagopa.wallet.domain.applications
+
+import java.time.Instant
+
+data class Application(
+    val id: ApplicationId,
+    val description: ApplicationDescription,
+    val status: ApplicationStatus,
+    val creationDate: Instant,
+    val lastUpdated: Instant
```

Comment:
```suggestion
    val lastUpdateDate: String,
```
You could call this field like that of a **WalletApplication**